views/publication/show.templ

```diff
@@ -9,157 +9,237 @@ import (
 									<div class="bc-toolbar-item">
-										@views.BadgeStatus(p.Status)
+										<button
+											class="btn btn-outline-danger"
+											hx-get={ c.PathTo("publication_confirm_withdraw", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
+											hx-target="#modals"
+										>
+											<i class="if if-arrow-go-back"></i>
+											<span class="btn-text">Withdraw</span>
+										</button>
 									</div>
-									if p.PublicationStatus != "" {
-										<div class="bc-toolbar-item">
-											<p class="c-subline text-nowrap pe-5 border-end">{ c.Loc.Get("publication_publishing_statuses." + p.PublicationStatus) }</p>
-										</div>
-									}
-									if p.Locked {
-										<div class="bc-toolbar-item" data-bs-toggle="tooltip" data-bs-title="Locked for editing" data-bs-placement="bottom">
-											<span class="c-subline text-nowrap pe-5 border-end">
-												<i class="if if-lock if--small if--muted"></i>
-												<span class="text-muted c-body-small ms-2">Locked</span>
-											</span>
-										</div>
-									}
+								}
+								if c.User.CanPublishPublication(p) && p.Status == "returned" {
 									<div class="bc-toolbar-item">
-										<span class="c-subline text-nowrap">
-											{ c.Loc.Get("publication_types." + p.Type) }
-											if p.Classification != "" {
-												{ p.Classification }
-											}
-										</span>
+										<button
+											class="btn btn-success"
+											hx-get={ c.PathTo("publication_confirm_republish", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
+											hx-target="#modals"
+										>
+											<div class="btn-text">Republish <span class="d-md-none d-lg-inline">to Biblio</span></div>
+										</button>
 									</div>
-								</div>
-								<div class="bc-toolbar-right">
-									<div class="bc-toolbar-item me-auto me-lg-0">
-										<div class="d-flex align-items-center flex-wrap justify-content-end">
-											<span class="c-subline text-truncate text-nowrap">
-												{ views.CreatedBy(c, p.DateCreated, p.Creator) }
-											</span>
-											<span class="c-subline text-truncate text-nowrap ps-5">
-												{ views.UpdatedBy(c, p.DateUpdated, p.User, p.LastUser) }
-											</span>
+								}
+								if c.User.CanPublishPublication(p) &&  p.Status != "returned" {
+									<div class="bc-toolbar-item">
+										<button
+											class="btn btn-success"
+											hx-get={ c.PathTo("publication_confirm_publish", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
+											hx-target="#modals"
+										>
+											<div class="btn-text">Publish <span class="d-md-none d-lg-inline">to Biblio</span></div>
+										</button>
+									</div>
+								}
+							</div>
+						</div>
+						<h4 class="w-100 mb-0 mb-lg-2">
+
+							if (p.Type == "issue_editor") || (p.Type == "book_editor") {
+								if len(p.Editor) == 1 && p.Editor[0] != nil {
```

Comment:
`nil` checks should not be necessary here (and on lines 167 and 171). They should never be `nil` as discussed [a while back](https://github.com/ugent-library/biblio-backoffice/commit/f068df91beec58fef14ca9d3812765ba5ec9f7ac#r144323945).